controllers/sms-gateway.js

```diff
@@ -112,6 +112,8 @@ module.exports = {
       getOutgoing
     ], (err, [,,outgoingMessages]) => {
       if (err) {
+        console.error('There was an error processing the following SMS POST request');
+        console.error(JSON.stringify(req.body));
```

Comment:
Are you sure you want to log request bodies?  Is there any worry about personally-identifying info with this?